src/ert/job_queue/queue.py

```diff
@@ -131,19 +129,19 @@ def __init__(self, driver: "Driver", max_submit: int = 2):
 
     @property
     def num_running(self) -> int:
-        return self._num_running()  # type: ignore
+        return self._num_jobs(JobStatusType.JOB_QUEUE_RUNNING)  # type: ignore
 
     @property
```

Comment:
Turns out that these are only used in `__str__` and `__repr__` could we just make those standard instead and remove these?

Reply:
Can you clarify what you mean with 'standard' in this context?

Reply:
I was thinking str the same as repr and repr satisfies https://docs.python.org/3/library/functions.html#repr in a simple way, such as "<JobQueue()>".